dxr/vcs.py

```diff
@@ -17,11 +17,12 @@
 - Check if the mercurial paths are specific to Mozilla's customization or not.
 
 """
-from datetime import datetime
```

Comment:
I probably shouldn't have removed this line, because it is the cause for the failing CI.